ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -1436,27 +1467,36 @@ def log_cpu_state_change(self, reachable, partial=False):
     def reachability_watcher(self):
         # This function watches the reachability of the CPU port, and ASIC. It logs the state
         # changes for future analysis
-        self.watcher_is_stopped.clear() # Watcher is running.
         while self.watching:
-            vlan_to_t1, t1_to_vlan = self.ping_data_plane(self.light_probe)
-            reachable              = (t1_to_vlan  > self.nr_vl_pkts * 0.7 and
+            # Data plane watcher:
+            if self.watch_dataplane:
+                self.watcher_dp_is_stopped.clear() # Dataplane Watcher is not stopped.
+                vlan_to_t1, t1_to_vlan = self.ping_data_plane(self.light_probe)
+                reachable              = (t1_to_vlan  > self.nr_vl_pkts * 0.7 and
                                       vlan_to_t1  > self.nr_pc_pkts * 0.7)
-            partial                = (reachable and
+                partial                = (reachable and
                                       (t1_to_vlan < self.nr_vl_pkts or
                                        vlan_to_t1 < self.nr_pc_pkts))
-            self.asic_flooding     = (reachable and
+                self.asic_flooding     = (reachable and
```

Comment:
Are you adding 'tab' here and in line 1477? please replace with white spaces.

Reply:
They are whitespaces.